.github/workflows/_build_test_upload.yml

```diff
@@ -228,7 +228,7 @@ jobs:
       - name: Install Dependencies
         run: |
           echo `python3 --version`
-          sudo apt-get install -y python-setuptools
```

Comment:
Just delete the whole line? You install setuptools later with pip anyway,

Reply:
trying here: https://github.com/pytorch/data/pull/1430